props/saber_sabersense_buttons.h

```diff
@@ -0,0 +1,1046 @@
+      STDOUT.println(dynamic_mixer.get_volume());
+    } else {
+      // Cycle through ends of Volume Menu option
+      #ifdef VOLUME_MENU_CYCLE
+        if (!max_vol_reached_) {
+          if (SFX_volmax) {
+            hybrid_font.PlayCommon(&SFX_volmax);
+          } else {
+            beeper.Beep(0.5, 3000);
+          }
+          STDOUT.print("Maximum Volume: ");
+          max_vol_reached_ = true;
+        } else {
+          dynamic_mixer.set_volume(std::max<int>(VOLUME * 0.1,
+          dynamic_mixer.get_volume() - VOLUME * 0.90));
+          if (SFX_volmin) {
+            hybrid_font.PlayCommon(&SFX_volmin);
+          } else {
+            beeper.Beep(0.5, 1000);
+          }
+          STDOUT.print("Minimum Volume: ");
+          max_vol_reached_ = false;
+        }
+      #else
+        if (SFX_volmax) {
+          hybrid_font.PlayCommon(&SFX_volmax);
+        } else {
+          beeper.Beep(0.5, 3000);
+        }
+        STDOUT.print("Maximum Volume: ");
+      #endif
+    }
+  }
+
+  void VolumeDown() {
+    STDOUT.println("Volume Down");
+    if (dynamic_mixer.get_volume() > (0.10 * VOLUME)) {
+      dynamic_mixer.set_volume(std::max<int>(VOLUME * 0.1,
+        dynamic_mixer.get_volume() - VOLUME * 0.10));
+      if (SFX_voldown) {
+        hybrid_font.PlayCommon(&SFX_voldown);
+      } else {
+        beeper.Beep(0.5, 2000);
+      }
+      STDOUT.print("Volume Down - Current Volume: ");
+      STDOUT.println(dynamic_mixer.get_volume());
+    } else {
+      #ifdef VOLUME_MENU_CYCLE
+        if (!min_vol_reached_) {
+          if (SFX_volmin) {
+            hybrid_font.PlayCommon(&SFX_volmin);
+          } else {
+            beeper.Beep(0.5, 1000);
+          }
+          STDOUT.print("Minimum Volume: ");
+          min_vol_reached_ = true;
+        } else {
+          dynamic_mixer.set_volume(VOLUME);
+          if (SFX_volmax) {
+            hybrid_font.PlayCommon(&SFX_volmax);
+          } else {
+            beeper.Beep(0.5, 3000);
+          }
+          STDOUT.print("Maximum Volume: ");
+          min_vol_reached_ = false;
+        }
+      #else
+        if (SFX_volmin) {
+          hybrid_font.PlayCommon(&SFX_volmin);
+        } else {
+          beeper.Beep(0.5, 1000);
+        }
+        STDOUT.print("Minimum Volume: ");
+      #endif
+      
+    }
+  }
+
+  //  Button Clicker to play press/release wav files when buttons are pressed.
+  //  Intended for Scavenger hilt where wheel makes tactile feel difficult.
+  //  Requires press.wav and release.wav files to work.
+    void PlaySound(const char* sound) {
+      RefPtr<BufferedWavPlayer> player = GetFreeWavPlayer();
+       if (player) {
+         if (!player->PlayInCurrentDir(sound)) player->Play(sound);
+        }
+      }            
+    bool Event2(enum BUTTON button, EVENT event, uint32_t modifiers) override {      
+      switch (EVENTID(button, event, modifiers)) {
+      case EVENTID(BUTTON_POWER, EVENT_PRESSED, MODE_ANY_BUTTON | MODE_ON):
```

Comment:
case statements should be indented more than the switch they are in.
(Every statement should be indented more than the block they are in.)

Reply:
Hopefully now sorted.